src/styles/utils/index.ts

```diff
@@ -792,17 +793,26 @@ function getBaseAutoCompleteSuggestionContainerStyle({left, bottom, width}: GetB
 /**
  * Gets the correct position for auto complete suggestion container
  */
-function getAutoCompleteSuggestionContainerStyle(itemsHeight: number): ViewStyle {
+function getAutoCompleteSuggestionContainerStyle(itemsHeight: number, platform: Platform, isMobileSafari: boolean): ViewStyle {
     'worklet';
 
+    // This if condition is reverting the workaround for broken scroll on all platforms but native android and iOS safari, where the issue with
+    // scrolling char behind suggestion list is occuring. Rewerting the fix resolves the issue with always scrollable list on other platforms.
     const borderWidth = 2;
-    const height = itemsHeight + 2 * CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTER_INNER_PADDING;
+    let height = itemsHeight + 2 * CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTER_INNER_PADDING;
+    let top = -(height + CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTER_PADDING);
+
+    if (platform === 'android' || isMobileSafari) {
+        top += borderWidth;
+    } else {
+        height += borderWidth;
+    }
```

Comment:
Check our [philosophy](https://github.com/Expensify/App?tab=readme-ov-file#philosophy) 5

> If there is a feature that simply doesn't exist on all platforms and thus doesn't exist in RN, rather than doing if (platform=iOS) { }, instead write a "shim" library that is implemented with NOOPs on the other platforms. For example, rather than injecting platform-specific multi-tab code (which can only work on browsers, because it's the only platform with multiple tabs), write a TabManager class that just is NOOP for non-browser platforms. This encapsulates the platform-specific code into a platform library, rather than sprinkling through the business logic.

Reply:
So in this case I should create two new files for iOS-Safari only, and android-native only?

Reply:
Yeah Android, web separately.

Reply:
In terms of separate web file, I will still have to determine if its mobile safari, or android chrome with `Browser.isMobileSafari();` is that the right approach?

Reply:
Yes, that's the way to go!

Reply:
Bump @MrRefactor

Reply:
updated @Santhosh-Sellavel